.travis.yml

```diff
@@ -1,5 +1,8 @@
 ---
+notifications:
+  email: false
 language: ruby
+sudo: false
 bundler_args: --without development
```

Comment:
add system_tests here too, cuz we're installing fog and all other things we don't need on travis